Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -241,7 +241,9 @@ class FakespotViewController: UIViewController, Themeable, Notifiable, UIAdaptiv
             contentStackView.addArrangedSubview(view)
 
             if let loadingView = view as? FakespotLoadingView {
-                loadingView.animate()
+                DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
+                    loadingView.animate()
```

Comment:
I know dispatching like this is awkward. Animation seems to start before the view is shown and that causes a nasty glitch.